.github/workflows/manifest.yml

```diff
@@ -28,8 +27,10 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+
       - name: Build AK
-        run: make bin/ak
+        run: go build -trimpath -o bin/ak ./cmd/ak
```

Comment:
Why?

Reply:
Noted in other PRs related to ENG-2014: `make bin/ak` uses timestamp-based flags which conflict with caching, so whenever we build AK in CI we use `go build`